src/app/app-routing.module.ts

```diff
@@ -25,6 +25,12 @@ const routes: Routes = [
         canActivate: [MsalGuard, RoleGuard],
         loadChildren: () => import('./features/search/search.module').then(m => m.SearchModule)
       },
+      {
+        path: 'create',
+        data: { title: 'Create new technical record', roles: Roles.TechRecordView },
```

Comment:
Is TechRecordView the correct role for a create path ? I see we have TechRecordCreate elsewhere ?